### PR TITLE
Closes Cyborg Module/Store on reset.

### LINKED ISSUE #15341: Tool list stays open when you get reset as a borg
**Issue Description**:
Tool list stays open when you get reset as a borg.

**What did you expect to happen**: 
Tool list should disappear.

**What happened instead**:
Tool list gets emptied but in the way cluttering the UI.

**Why is this bad/What are the consequences**:
The button to toggle the visibility of the tool list is the same as the one to pick a module. Making it impossible to close the tool list without picking a module first.

**Steps to reproduce the problem**:
1. Be a borg
2. Pick a module
3. Open tool list
4. Be reset

**When did the problem start happening**:
Unknown

**Extra information**:
Here is a screenshot:
bug3

### PR DESCRIPTION
**Pull request by CkeyLurapa**
**## What Does This PR Do**
Closes Robot Module, and Store on reset.

**## Why It's Good For The Game**
Makes the resetting of Cyborgs less of a UI hickup. Allows Cyborgs who left their module open, to 1 click select a module. Versus closing a blank module, and selecting a module.

**## Images of changes**

a2SrQE9MFh

**## Changelog**
:cl:
fix: Cyborgs when reset now have their module UI closed.
/:cl:
Fixes #15341

### CODE REVIEW

**Comment by CkeyLurapa** [comment]
Ignore the commits, besides 'Update robot_upgrades.dm' circled in the image below was learning Github. And was using my main, not its branches. So I had to revert my main, and can't figure out how to delete the commits history even though it was reverted.

code for pr
oops

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 1]
```diff
@@ -30,8 +30,10 @@
 /obj/item/borg/upgrade/reset/action(mob/living/silicon/robot/R)
 	if(..())
 		return
-
 	R.reset_module()
+	R.shown_robot_modules = 0
+	R.client.screen -= R.robot_modules_background
+	R.client.screen -= R.hud_used.module_store_icon
```
Comment (SteelSlayer) [inline code/game/objects/items/robot/robot_upgrades.dm:36]: These three lines should probably go into the `reset_module()` proc now that I think about it, just so we can call that proc from anywhere and still have the store icon removed.

Comment (AffectedArc07) [reply code/game/objects/items/robot/robot_upgrades.dm:36]: Put these into the `reset_module()` proc.

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]
Putting a 7-day time on this for you to make the changes or have the PR closed.

**Comment by AffectedArc07** [comment]
Closed after 10 days of nothing

**Pull request closed without merging**